src/main/java/frc/robot/swerve/SwerveConfig.java

```diff
@@ -58,48 +49,32 @@ public class SwerveConfig {
     public static final int BR_TURN_CANID = 20;
     public static final boolean BR_REVERSE = false;
 
-    // helper function
-
-
-
-
-
-
-    ///////////////////////////////////////////// UNDERCONSTRUCTION
-
-
-
-
-    //The createDriveMotor method 
-
-    public static void createDriveMotor(int canld, boolean reverseMotor){
-
-        TalonFX DriveMotor = new TalonFX(canld);
-        {
-          DriveMotor.configSelectedFeedbackSensor(FeedbackDevice.IntegratedSensor);
-          DriveMotor.configSoftLimitDisableNeutralOnLOS(true, canld);
-          DriveMotor.setInverted(InvertType.None);
-          DriveMotor.setNeutralMode(NeutralMode.Brake);
-          DriveMotor.configClosedloopRamp(0.3);
-          DriveMotor.config_kP(0, 0.1);
-          DriveMotor.config_kI(0, 0.001);
-          DriveMotor.config_kD(0, 5.0);
-          DriveMotor.config_kF(0, 1023.0 / 20660.0);
-          DriveMotor.config_IntegralZone(0, 300.0);
-          DriveMotor.configMaxIntegralAccumulator(0, 1.0);
+    /**
+     * Creates a drive motor for the swerve drive. Drive motors are used with
+     * velocity-based PID control.
+     */
+    public static TalonFX createDriveMotor(int canId, boolean reverseMotor){
+        TalonFX driveMotor = new TalonFX(canId);
```

Comment:
I changed `canld` to `canId`. It can be hard to distinguish a lowercase 'L' from an uppercase 'I'. Sorry I wasn't more clear in my email.

I also changed `DriveMotor` to `driveMotor`. Capitalization is a weird culture thing in programming - people using different languages typically have different traditions. You can read about the different standards [here](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/).

In Java, we typically use what's called `camelCase` for variable and method names - first letter is lowercase, subsequent words separated with a capital letter. We use `PascalCase` for class names, and we use `UPPER_SNAKE_CASE` for public constant values (like the ones at the top of this class).

Won't break anything, just useful to keep consistent.